module-services/service-appmgr/model/ApplicationManagerCommon.cpp

```diff
@@ -750,8 +754,9 @@ namespace app::manager
             actionsRegistry.finished();
             break;
         default: {
-            auto &params = action->params;
-            app::ApplicationCommon::requestAction(this, app.name(), action->actionId, std::move(params));
+            // IMO this is not neccessary! Commenting for now:
+            // auto &params = action->params;
+            // app::ApplicationCommon::requestAction(this, app.name(), action->actionId, std::move(params));
```

Comment:
you do not need to store code in comments in history - you are sure things can be removed? remove them. comments do not compile